### PR TITLE
Maintenance of IBP endpoints for several chains

### PR DESCRIPTION
**Pull request by miloskriz**
Dear team, hello, 

In this request please consider the update of endpoints for serving the following chains:

- `coretime-kusama`
- `coretime-westend`
- `people-westend`
- `asset-hub-paseo`

All these endpoints are operated by the Infrastructure Builders Programme under a GeoDNS decentralisation strategy.

Many thanks!!

**_Milos_**

### CODE REVIEW

File: packages/apps-config/src/endpoints/productionRelayKusama.ts [hunk 1]
```diff
@@ -945,6 +945,8 @@ export const prodParasKusamaCommon: EndpointOption[] = [
     info: 'kusamaCoretime',
     paraId: 1005,
     providers: {
+      // 'IBP-GeoDNS1': 'wss://sys.ibp.network/coretime-kusama',
```
Comment (TarikGul) [inline packages/apps-config/src/endpoints/productionRelayKusama.ts:948]: Can we add this PR link to the right of each of these as an additional comment like: `// https://github.com/polkadot-js/apps/pull/10514/files`

That way we know why these are commented out.

**Review by TarikGul — CHANGES_REQUESTED** [review]
Just one nit above.

**Comment by miloskriz** [comment]
Dear @TarikGul, 

Many thanks for your message, please note that we have since deployed the proposed `coretime-kusama` endpoints in full and thus the lines do not need to be commented out anymore.. 

What do you think?

Thanks again!! and best regards!!

**_Milos_**

File: packages/apps-config/src/endpoints/productionRelayKusama.ts (hunk 1 above)
Comment (miloskriz) [reply packages/apps-config/src/endpoints/productionRelayKusama.ts]: Not needed anymore!! the endpoints are live now 😃 

Many thanks

**Review by miloskriz — COMMENTED** [review]
Done!

**Review by TarikGul — APPROVED** [review]

**Comment by TarikGul** [comment]
Just some conflicts need to be resolved and I'll merge it in first, thanks for the patience!

**Pull request merged**